docs/user/project_metadata.md

```diff
@@ -0,0 +1,106 @@
+# Project Metadata
+
+
+## Project-URL
```

Comment:
Nitpick: `Project-URL` is the metadata name at distribution bake time, but most users probably know it better as the `[project.urls]` table. I think we should make the title here just "Project URLs" and link to the appropriate part of the `pyproject.toml` spec, rather than the core metadata spec.

```suggestion
## Project URLs
```